twitter.html

```diff
@@ -0,0 +1,333 @@
+          <!-- twitter button header -->
+          <button class="header-btn">Tweet</button>
+        </header>
+
+        <!-- MAIN  -->
+
+        <main class="main">
+          <div>
+            <img
+              src="images/pitures/pexels-markus-winkler-4160060.jpg"
+              class="pic-profile-2"
+              alt="Picture"
+            />
+          </div>
+          <p class="firt-tweet">
+            <ion-icon name="tennisball-outline"></ion-icon>
+            <span class="gry-font">
+              Lorem ipsum dolor sit amet consectetu
+            </span>
+            <br />
+            <strong>Lorem ipsum dolor</strong
+            ><span class="gry-font-big">@consectetu 26m</span> <br />Lorem ipsum
+            dolor sit amet consectetur, adipisicing elit. Officia, doloribus? Nu
+            <ion-icon name="stats-chart-outline" class="ylw-clr"></ion-icon>
+            <br /><br />
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti
+            nihil consequatur dolorum tenetur fuga
+            <ion-icon name="storefront-outline" class="clr-grn"></ion-icon>
+            <br />
+            <br />
+            <ion-icon
+              name="image-outline"
+              class="space-btn-out list"
+            ></ion-icon>
+            <ion-icon name="gift-outline" class="space-btn-out list"></ion-icon>
+            <ion-icon
+              name="bar-chart-outline"
+              class="space-btn-out list"
+            ></ion-icon>
+            <ion-icon
+              name="happy-outline"
+              class="space-btn-out list"
+            ></ion-icon>
+            <ion-icon
+              name="calendar-clear-outline"
+              class="space-btn-out"
+            ></ion-icon>
+          </p>
+          <div>
+            <ion-icon name="ellipsis-horizontal-outline"></ion-icon>
+          </div>
+          <div>
+            <img
+              src="images/pitures/pexels-markus-winkler-4160060.jpg"
+              class="pic-profile-2"
+              alt="Picture"
+            />
+          </div>
+          <div>
+            <p>
+              <strong>electus aperiam </strong>
+              <span class="gry-font-big"></span> <br />
+              Lorem ipsum
+              <span class="blue-hash-tags"> #voluptates </span> dolor sit amet
+              adipisicing elit. Et,<span class="blue-hash-tags">
+                #consectetur
+              </span>
+              beatae, dignissimos delectus aperiam excepturi, pariatur nam vero
+            </p>
+            <br />
+            <a href="#">Read More:hia.jsabfu47914</a>
+            <img
+              src="images/pitures/pexels-tina-simakova-10288856 (1).jpg"
+              alt="picture"
+              class="main-pic"
+            />
+          </div>
+        </main>
+      </div>
+
+      <!--  RIGHT MENU -->
+
+      <div class="right-menu">
+        <div class="search-bar">
+          <p class="search-bar">Search Twitter</p>
```

Comment:
This should be a `<input type="text" />` rather than a `<p>`. The input element will give you a text box that will allow the user to type into to search twitter